_posts/2017/2017-04-05-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "JavaLand 2017 - Welcome to Phantasialand"
+date: "2017-04-05 15:16:17"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin, Bastian"]
+---
+
+The [JavaLand 2017](https://www.javaland.eu/en/javaland-2017/) conference took place for the 4th time in the famous [Phantasialand](http://www.phantasialand.de/en/) holiday park in Brühl, near Cologne. The program consisted of [talks](https://programm.javaland.eu/2017/schedule.html) on the frist two days and [in-depth workshops](https://www.javaland.eu/de/programm/schulungstag) at the last and third day.
+
+Here are some random facts so that you can imagine the dimension of the largest community conference for Java in Europe:
```

Comment:
Here are some random facts to get an impression on the dimension of the largest..."